plugins/flytekit-inference/setup.py

```diff
@@ -0,0 +1,38 @@
+from setuptools import setup
+
+PLUGIN_NAME = "inference"
+
+microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
+
+plugin_requires = ["flytekit>=1.13.0,<2.0.0", "kubernetes", "openai"]
```

Comment:
the plugin doesn't really require `openai`, right?

Reply:
> what sort of inference plugins we're going to add to flytekitplugins-inference
I've got ollama working with the sidecar pattern, so will be adding that next. in `flytekitplugins-inference`, i believe we need to stick to serving plugins that can be implemented with the sidecar pattern. if not, they should be standalone plugins. 

this parent plugin should essentially capture inference providers that allow self-hosting and serving models within a flyte task. the ideal goal is to make it work with actors to enable near real-time inference.

 > the plugin doesn't really require openai, right?
you're right. however, since most inference providers now support the openai api, it'd be beneficial to include it as part of the plugin.

Reply:
@eapolinario